tools/aie2xclbin/XCLBinGen.cpp

```diff
@@ -299,17 +299,13 @@ static LogicalResult generateCoreElfFiles(ModuleOp moduleOp,
         std::string targetLower = StringRef(TK.TargetArch).lower();
         SmallVector<std::string, 10> flags;
         flags.push_back("-O2");
-        std::string targetFlag = "--target=" + targetLower + "-none-elf";
+        std::string targetFlag = "--target=" + targetLower;
```

Comment:
Should still have the 'none-elf' added here or it won't pick up the clang-rt IIRC.

Reply:
It looks in the wrong place for the builtins if I leave this as is:
`ld.lld: error: cannot open C:\s\llvm-aie\install\lib\clang\17\lib\libclang_rt.builtins-aie2.a: no such file or directory
clang: error: ld.lld command failed with exit code 1 (use -v to see invocation)`
The correct location is: `C:\s\llvm-aie\install\lib\clang\17\lib\aie2\libclang_rt.builtins.a`

Reply:
The correct location should include the full triple "aie2-unknown-none-elf".  So, you need to have the full target triple as the target when you're building compiler-rt (and libc).  This path seems to indicate that that didn't happen...

Reply:
The directions I got from Konstantin don't use the full triple, iirc.